lecture_1/homework/app/models/Magical.rb

```diff
@@ -0,0 +1,9 @@
+class Magical < Weapon
+    def initialize
```

Comment:
To niestety nie zadziała. W modelach ActiveRecord nie możemy nadpisywać `initialize` w taki sposób. Jeżeli chcesz ustawić wartości domyślne to możesz skorzystać na przykład z callbacków lub railsowych [attributes](https://api.rubyonrails.org/classes/ActiveRecord/Attributes/ClassMethods.html)

Reply:
Dzięki za zwrócenie uwagi! Poprawione na attributes.